### DISCUSSION TITLE
MQTT Frontend Address

### DISCUSSION THREAD

**Discussion by adhic86** [Koenkk/zigbee2mqtt#21433] [General]
Hi

I'm a total newbie at this, so I apologise in advance if I'm not explaining properly. I've set up MQTT on my HomeBridge device successfully; when connecting via the MQTT Explorer app, it successfully connects. The MQTT Explorer is connecting via MQTT://192.168.0.202:1883. However, I can't access the Frontend page. Where can I find the address to access this? I've tried 192.168.0.202:1883, 192.168.0.202:8080 & 192.168.0.202:8082, but no luck. Can someone please help?

**Comment by Budgie66** [comment]
the frontend on mine is 192.168.0.***:8081

**Reply by adhic86** [reply]
Thank you, unfortunately I still cannot access the Frontedge page

**Comment by chris-1243** [comment]
Would you provide your configuration.yaml and the log of Z2M (debug log)

**Reply by adhic86** [reply]
I’ve been able to reinstall everything & I think I’ve isolated the issue. If I try to connect via MQTT Explorer the status is offline, so maybe that’s the issue? If so can you offer any advice please?

**Reply by chris-1243** [reply]
Without any log it is hard to point where an error could be.

Is your zigbee coordinator (usb, lan) accessible to Zigbe2mqtt?

Is Z2M able to connect to your mqtt server? Check username/password...
The main problems at starting Z2M come from those usually.

**Reply by adhic86** [reply]
Thanks Chris. Could you tell me what you need & how I do it please?

**Reply by chris-1243** [reply]
Normally, your instance of Z2M should create a log file.

What OS / support do you use? (Docker, haos,...?)
